.gitmodules

```diff
@@ -49,3 +49,9 @@
 [submodule "platform/windows/vendor/vcpkg"]
 	path = platform/windows/vendor/vcpkg
 	url = https://github.com/microsoft/vcpkg.git
+[submodule "vendor/harfbuzzFreetype/freetype"]
+	path = vendor/harfbuzzFreetype/freetype
+url=https://github.com/freetype/freetype.git
+[submodule "vendor/harfbuzzFreetype/harfbuzz"]
+	path = vendor/harfbuzzFreetype/harfbuzz
+url=https://github.com/alanchenboy/harfbuzz
```

Comment:
Nice that CI (almost) passes!

Let me know when you feel this is ready for review.

Can you get your changes upstreamed?

Reply:
Rollback to https://github.com/harfbuzz/harfbuzz 
Need to update qt runner's cmake to 3.12

Reply:
And needn't review it now, I think I will operate with @wipfli on how to change style things, once the style define is done, and then I believe this PR will be ready for review.

Reply:
I think we can review now.